pycqed/instrument_drivers/meta_instrument/qubit_objects/mock_CCL_Transmon.py

```diff
@@ -60,164 +62,284 @@ def add_mock_params(self):
+                                          self.mock_freq_res_bare()),
                            parameter_class=ManualParameter)
 
         self.add_parameter('mock_chi', label='dispersive shift', unit='Hz',
                            initial_value=self.mock_chi01()-self.mock_chi12()/2,
                            parameter_class=ManualParameter)
 
-        self.add_parameter('noise', label='noise level', unit='V',
-                           initial_value=0.01e-3, parameter_class=ManualParameter)
+        self.add_parameter('mock_freq_res', label='resonator frequency',
+                           unit='Hz', parameter_class=ManualParameter,
+                           initial_value=self.mock_freq_res_bare() -
+                           self.mock_chi12() / 2)
 
-        self.add_parameter('mock_res_width', label='resonator peak width', unit='Hz',
-                           initial_value=1e6, parameter_class=ManualParameter)
+        self.add_parameter('mock_res_width', label='resonator peak width',
+                           unit='Hz', initial_value=1e6,
+                           parameter_class=ManualParameter)
 
-        self.add_parameter('mock_flux_sensitivity', label='sensitivity to flux in current',
+        self.add_parameter('mock_flux_sensitivity',
+                           label='sensitivity to flux in current',
                            unit='A',
-                           initial_value={'FBL_1': 10e-3, 'FBL_2': 1},
+                           initial_value={'FBL_1': 1, 'FBL_2': 0.01},
                            parameter_class=ManualParameter)
 
-        self.add_parameter('mock_fl_dc_ch', label='most closely coupled fluxline',
-                           unit='', initial_value='FBL_1', parameter_class=ManualParameter)
+        self.add_parameter('mock_flux_current_overall', unit='A',
+                           initial_value=12e-3,
+                           parameter_class=ManualParameter)
 
-        self.add_parameter('resonator_freqs', unit='Hz', initial_value=None,
+        self.add_parameter('mock_cfg_dc_flux_ch',
+                           label='most closely coupled fluxline',
+                           unit='', initial_value='FBL_1',
                            parameter_class=ManualParameter)
 
-    def find_resonators_VNA(self, start_freq=7e9, stop_freq=8e9, power=-40,
-                            bandwidth=200, timeout=200, npts=50001, name='VNA'):
-        VNA = self.instr_VNA.get_instr()
-        t_start_meas = ma.a_tools.current_timestamp()
+        self.add_parameter('noise', label='nominal noise level', unit='V',
+                           initial_value=0.16e-3, parameter_class=ManualParameter)
 
-        VNA.start_frequency(start_freq)
-        VNA.stop_frequency(stop_freq)
-        VNA.power(power)
-        VNA.bandwidth(bandwidth)
-        VNA.npts(npts)
-        # VNA.timeout(timeout)
+    def find_resonators(self, start_freq=7e9, stop_freq=8e9, power=-40,
```

Comment:
The find method should not be defined in the mock qubit, instead the measure should be defined in the mock and the routine making use of it defined in the qubit_object abstract base class.